default.nix

```diff
@@ -55,6 +55,7 @@ let
   llvm-backend = callPackage ./nix/llvm-backend.nix {
     inherit llvmPackages src;
     inherit (ttuegel) cleanSourceWith;
+    host.clang = clang;
```

Comment:
What is `host`?

Reply:
The usual terminology in compilers is:

- `build` is the machine that is building the compiler
- `host` is the machine where the compiler will run
- `target` is the machine where the programs _built_ by the compiler will run (you usually only see this with cross-compiling)

This is the `host` Clang because it will run when the LLVM backend runs, but it won't be used to build the backend itself.